src/Sarif/Readers/DelegatingStream.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    internal class DelegatingStream : Stream
+    {
+        private readonly Stream stream;
+
+        /// <summary>
+        /// This is a wrapper for a stream that protects the underlying stream from being disposed.
+        /// Disposing this object is a no-op. To dispose the underlying stream use the DisposeUnderlyingStream method.
```

Comment:
>DisposeUnderlyingStream [](http://example.com/codeflow?start=87&length=23)

in a 'real' doc comment, this would be a reference to the method. but we don't need that, as we're not exhaustively documenting this type.